src/main/api/2/studio-api-2.yaml

```diff
@@ -4,7 +4,7 @@ servers:
 info:
   description: |-
     Crafter Studio API v2
-  version: '3.1.4.3'
+  version: '3.1.4.4'
```

Comment:
Please check the latest version of API. I think it is already at 3.1.4.4 if not higher